lib/repos.js

```diff
@@ -102,32 +196,167 @@ Repos.prototype.read = function (callback) {
   }, []);
 
   async.map(lookups, function (obj, cb) {
```

Comment:
`cb` is the convention for the `async` lib when you have a higher level `callback` for the function as a whole which I have.

`next` is almost never used with `async`.